mllib/src/test/scala/org/apache/spark/mllib/evaluation/MultilabelMetricsSuite.scala

```diff
@@ -47,7 +47,7 @@ class MultilabelMetricsSuite extends SparkFunSuite with MLlibTestSparkContext {
     val scoreAndLabels: RDD[(Array[Double], Array[Double])] = sc.parallelize(
       Seq((Array(0.0, 1.0), Array(0.0, 2.0)),
         (Array(0.0, 2.0), Array(0.0, 1.0)),
-        (Array(), Array(0.0)),
+        (Array.empty[Double], Array(0.0)),
```

Comment:
Normally would say don't bother with this either but I know you just cleaned up a lot of this for consistency. I'm neutral about whether it's more readable and it's only test code.